akka-cluster/src/main/resources/reference.conf

```diff
@@ -289,5 +289,10 @@ akka {
       adaptive-group = "akka.cluster.routing.AdaptiveLoadBalancingGroup"
     }
   }
+  
+  coordinated-shutdown-phases {
+    cluster-pre-exiting = []
+    cluster-exiting = [cluster-pre-exiting]
+  }
```

Comment:
We could define such default phases, but then they must be defined in reference.conf of akka-actor. It's fine to not add any tasks to a phase, then it will be completed immediately.